### ISSUE TITLE
NullPointerException in Telegram

### ISSUE THREAD

**Issue by g4s8** [zerocracy/farm#1014]
NullPointerException in `ReProfile$MsgText.toString` https://sentry.io/zerocracy/farm/issues/555695078/
```
java.lang.NullPointerException: null
    at com.zerocracy.radars.telegram.ReProfile$MsgText.toString(ReProfile.java:87)
    at com.zerocracy.radars.telegram.ReProfile.react(ReProfile.java:47)
    at com.zerocracy.radars.telegram.ReSafe.lambda$react$0(ReSafe.java:64)
    at org.cactoos.func.FuncWithFallback.apply(FuncWithFallback.java:84)
    at org.cactoos.func.IoCheckedFunc.lambda$apply$0(IoCheckedFunc.java:59)
    at org.cactoos.scalar.IoCheckedScalar.value(IoCheckedScalar.java:70)
    at org.cactoos.func.IoCheckedFunc.apply(IoCheckedFunc.java:59)
    at com.zerocracy.radars.telegram.ReSafe.react(ReSafe.java:99)
    at com.zerocracy.radars.telegram.TmZerocrat.onUpdateReceived(TmZerocrat.java:92)
    at java.util.ArrayList.forEach(ArrayList.java:1257)
    at org.telegram.telegrambots.generics.LongPollingBot.onUpdatesReceived(LongPollingBot.java:27)
    at org.telegram.telegrambots.updatesreceivers.DefaultBotSession$HandlerThread.run(DefaultBotSession.java:311)
```

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by yegor256** [comment]
@g4s8 fixed already, thanks: e84075084417ce55d0df77c0a29455a086a3bbaa

**Comment by 0crat** [comment]
The job is not in WBS, won't close the order
